docs/Sources/tuist/tuist.docc/Articles/Tuist Cloud/Users/tuist-cloud.md

```diff
@@ -22,6 +22,12 @@ Tuist Cloud offers a robust storage solution for Tuist, enabling the sharing of
 
 > Tip: To assist organizations in evaluating their return on investment (ROI), we've developed an [**ROI calculator**](https://tuist.io/cloud). For instance, consider an organization with approximately 20 developers. If their clean builds take 10 minutes and they achieve a 70% cache effectiveness, they could potentially reduce development time by 24,000 hours and recover up to $6.4 million a year.
 
+#### Selective testing across environments
+
+Once teams reach a certain scale, they often grapple with optimizing their CI process to maintain quick turnaround times. While **testing everything** continually might work for smaller teams, it becomes impractical on a larger scale. At this juncture, many teams resort to investing in superior hardware, creating custom tools, complicating their CI pipelines, or worse, accepting slower development cycles. But there's a better way.
```

Comment:
What do you think if we mention that: **we make test execution incremental across environments**.